composer.json

```diff
@@ -15,8 +15,8 @@
         "sabberworm/php-css-parser": "8.1.1"
     },
     "require-dev": {
-        "wp-coding-standards/wpcs": "1.2.1",
+        "wp-coding-standards/wpcs": "^1",
         "dealerdirect/phpcodesniffer-composer-installer": "0.4.4",
-        "wimg/php-compatibility": "8.2.0"
```

Comment:
I think these changes can be reverted, because:

> It furthermore ensures that they stay up to date within non-breaking versions, with the concrete versions of course continuing to be managed with `composer.lock` (run `composer install` to install the tested versions and `composer update` to potentially update).

This is going to be handled by Renovate going forward. You can see a bunch of PRs that Renovate has open right now for bumping these versions: https://github.com/ampproject/amp-wp/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+renovate

Reply:
Yes, all changes that modify versions can be reverted since Renovate will handle them.